framework/src/postprocessors/ElementExtremeValue.C

```diff
@@ -50,11 +55,11 @@ ElementExtremeValue::initialize()
   switch (_type)
   {
     case MAX:
-      _value = -std::numeric_limits<Real>::max(); // start w/ the min
+      _proxy_value = -std::numeric_limits<Real>::max(); // start w/ the min
```

Comment:
Initialize both `_value` and `_proxy_value` here, below, and also in the nodal object